sample-operator/README.md

```diff
@@ -38,11 +38,11 @@ $ kubectl get pod -l app=sample-operator
 NAME                              READY     STATUS    RESTARTS   AGE
 sample-operator-821691060-m5vqp   1/1       Running   0          3m
```

Comment:
maybe we can nest this in the Start the Operator section so folks can verify it's running before creating the CRD

Reply:
good idea, will move it up